site/content/en/docs/tasks/dev/integrate_a_custom_job.md

```diff
@@ -92,6 +93,16 @@ Here are completed external integrations you can learn from:
 Add your framework's GroupVersionKind to `.integrations.externalFrameworks` in [controller_manager_config.yaml](
 https://kueue.sigs.k8s.io/docs/installation/#install-a-custom-configured-released-version).
 
+### RBAC Augmentation
+
+Kueue will need permission to get, list, and watch instances of your CRD.
+
+If you are building a custom Kueue deployment, simply add a `kubebuilder:rbac` annotation to the source code
+(for example in integrationmanager.go) and regenerate the manifests.
+
+If you are deploying a Kueue release, modify either charts/kueue/templates/rbac/role.yaml or
```

Comment:
Better to have link for these files for quick reference.

Reply:
Done.